ferrocene/tools/self-test/src/targets.rs

```diff
@@ -17,6 +17,9 @@ static SUPPORTED_TARGETS: &[TargetSpec] = &[
         linker: Linker::CrossCC(&["aarch64-linux-gnu-"]),
     },
     TargetSpec { triple: "aarch64-unknown-none", std: false, linker: Linker::BundledLld },
+    TargetSpec { triple: "armv8r-none-eabihf", std: false, linker: Linker::BundledLld },
```

Comment:
I'm pretty sure this should be like:

```
    TargetSpec { triple: "armv8r-none-eabihf", std: false, linker: Linker::CrossCC(&["arm-none-eabi-"]) },
    TargetSpec { triple: "thumbv7em-none-eabihf", std: false, linker: Linker::CrossCC(&["arm-none-eabi-"]) },
    TargetSpec { triple: "thumbv7em-none-eabi", std: false, linker: Linker::CrossCC(&["arm-none-eabi-"]) },
```

However I'm unable to get this to work locally thus far...

Reply:
Hmm, why? They should be using the bundled LLD.

Reply:
Okay, I started questioning myself!